jwst/scripts/collect_pipeline_cfgs.py

```diff
@@ -1,54 +1,63 @@
 #!/usr/bin/env python
 
-import os
 import sys
+import logging
+from pathlib import Path
 
 from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
 
 
+# Set logger to only print to screen
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+stream_handle = logging.StreamHandler()
+stream_handle.setLevel(logging.INFO)
+logger.addHandler(stream_handle)
```

Comment:
This appears to go beyond just a style change, what was the impetus for making this change?  Why is it necessary here but not in other scripts?

Reply:
Ruff is complaining about the print statements, so I added this to remove them and still print to screen.